plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -226,6 +226,7 @@ func defaultPriorities() sets.String {
 		factory.RegisterPriorityFunction2("NodeAffinityPriority", priorities.CalculateNodeAffinityPriorityMap, priorities.CalculateNodeAffinityPriorityReduce, 1),
 
 		// TODO: explain what it does.
+		//Taints are some restricts added to Nodes,only pod tolerates it can be schedule onto those Nodes.
```

Comment:
`Taints are restrictive markers added to Nodes. Only Pods which explicitly tolerate a particular taint will be scheduled onto a Node marked with that taint.`